include/dictionary-search.php

```diff
@@ -269,11 +273,15 @@ function get_subquery_where()
 	}
 	$search = strtolower($search);
 
-	$key = $_GET['key'];
-	if(!isset($key))
```

Comment:
This logic was backward. We need to check if `$_GET['key'])` is set before we try to read its value. This was causing problems in the unit tests.